app/gui2/src/util/__tests__/array.test.ts

```diff
@@ -14,18 +14,20 @@ fcTest.prop({
 })
 
 fcTest.prop({
-  arr: fc
-    .array(fc.float())
-    .map((a) => ({ arr: a.sort((a, b) => a - b), i: Math.floor(Math.random() * a.length) })),
+  arr: fc.array(fc.float({ noNaN: true })).map((a) => ({
```

Comment:
I think instead of opting out of testing NaNs, maybe we should actually handle them properly. In this case, it would be by modifying the predicate to `(n) => !isNaN(n) && n < target`

Reply:
just did a little test... it turns out that sorting doesn't work well with `NaN` anyway: `[3, NaN, 3, 1].sort((a, b) => a - b)` results in `[3, NaN, 1, 3]` so in this case i think it's fine to just avoid `NaN`

Reply:
Okay I see. In that case let's avoid it.